packages/core/src/codewhisperer/ui/codeWhispererNodes.ts

```diff
@@ -29,21 +29,21 @@ import { submitFeedback } from '../../feedback/vue/submitFeedback'
 import { focusAmazonQPanel } from '../../codewhispererChat/commands/registerCommands'
 import { isWeb } from '../../shared/extensionGlobals'
 
-export function createAutoSuggestions(pause: boolean): DataQuickPickItem<'autoSuggestions'> {
+export function createAutoSuggestions(running: boolean): DataQuickPickItem<'autoSuggestions'> {
     const labelResume = localize('AWS.codewhisperer.resumeCodeWhispererNode.label', 'Resume Auto-Suggestions')
     const iconResume = getIcon('vscode-debug-start')
     const labelPause = localize('AWS.codewhisperer.pauseCodeWhispererNode.label', 'Pause Auto-Suggestions')
     const iconPause = getIcon('vscode-debug-pause')
 
     return {
         data: 'autoSuggestions',
-        label: pause ? codicon`${iconPause} ${labelPause}` : codicon`${iconResume} ${labelResume}`,
-        description: pause ? 'Currently RUNNING' : 'Currently PAUSED',
+        label: running ? codicon`${iconPause} ${labelPause}` : codicon`${iconResume} ${labelResume}`,
+        description: running ? 'Currently RUNNING' : 'Currently PAUSED',
```

Comment:
It looks like the logic was "swapped", i.e. it treated `pause` as if it were `running`. So I renamed the param to `running`. @hayemaxi

Reply:
what does `running` mean? Why when running is true, the label is pause?

Reply:
It's the same as the previously-existing `pause` parameter, which controls what this function returns.

> what does running mean? Why when running is true, the label is pause?

)

- The "play" button shows when it's paused. 
- The "pause" button shows when it's running.
- 
![image](https://github.com/user-attachments/assets/1c4441aa-95f6-4051-a306-fe416ff8fc2f